auctioneer/program/src/execute_sale/mod.rs

```diff
@@ -221,5 +224,25 @@ pub fn auctioneer_execute_sale<'info>(
 
     invoke_signed(&ix, &cpi_accounts.to_account_infos(), &[&auctioneer_seeds])?;
 
+    let close_listing_config = listing_config.lamports();
```

Comment:
So I took a look at this and the `listing_config` is an `Account` (https://docs.rs/anchor-lang/0.25.0/anchor_lang/accounts/account/struct.Account.html) which is just a wrapper around `AccountInfo`, which does have a `lamports` field. To access that `AccountInfo` you need to call `to_account_info()` to unwrap `listing_config`.